### PR TITLE
Updating Get Involved page

### PR DESCRIPTION
**Pull request by pedrofs**
This pull request updates the Get Involved page with the latest page provided by hibaldo (cake-site-12.zip)

Please, take special attention to the links I added to `config/site.php` to ensure they are right.

### CODE REVIEW

File: config/site.php [hunk 1]
```diff
@@ -115,12 +115,16 @@
 						'title' => __('Forum'),
 						'options' => ['target' => '_blank'],
 					],
+					'quora' => [
+						'url' => 'https://www.quora.com/topic/CakePHP',
```
Comment (markstory) [inline config/site.php]: Do people still use quora? I've personally never found it helpful ever.

File: config/site.php [hunk 2]
```diff
@@ -135,6 +139,16 @@
 						'url' => 'http://www.cakedc.com/',
 						'title' => __('Paid Support')
 					],
+					'googleplus' => [
+						'url' => 'https://plus.google.com/communities/108328920558088369819',
+						'options' => ['target' => '_blank'],
+						'title' => __('Google+')
+					],
+					'googlegroup' => [
+						'url' => 'https://groups.google.com/forum/#!forum/cake-php',
```
Comment (markstory) [inline config/site.php]: We've shutdown/archived the google group.

File: src/Template/Element/get-involved/get-help.ctp [hunk 3]
```diff
@@ -114,6 +140,19 @@ use Cake\Core\Configure;
...
+		</div>
+		<div class="col-sm-11 p0">
+			<?= $this->Html->tag(
+				'p',
+				__('{0}: Official discussion group for the project',
+					$this->Html->link(__('Google Group'), Configure::read('Site.menu.items.help.googlegroup.url'), ['target' => '_blank'])
+				))
+			?>
+		</div>
+	</div>
```
Comment (markstory) [inline src/Template/Element/get-involved/get-help.ctp]: I don't think we should be promoting a read only archive.

**Comment by pedrofs** [comment]
Quora and google group removed.

Thanks for the your @markstory.

**Pull request merged**